<launch>

  <include file="$(find staubli_barretthand_moveit_config)/launch/demo.launch"/>

  <param name="moveGroupName" value="StaubliArm" />
  <node name="moveit_trajectory_planner" pkg="moveit_trajectory_planner" type="moveit_trajectory_planner_node.py" output="screen"/>
  <test test-name="moveit_trajectory_planner_test" name="test_node" pkg="moveit_trajectory_planner" type="staubli_moveit_trajectory_planner_test_node.py" />

</launch>